docs/deploy-server.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+const express = require('express');
+
+const app = express();
+
+const port = process.env.PORT ? process.env.PORT : 3000;
+const dist = path.join(__dirname, '_site');
+
+app.use('/images', express.static(path.join(__dirname, 'images')));
+app.use('/', express.static(dist));
+
+
+app.listen(port, (error) => {
+  if (error) {
+    console.log(error); // eslint-disable-line no-console
+  }
+  console.info('Express is listening on port %s.', port); // eslint-disable-line no-console
```

Comment:
This should probably be an `if-else` block.
Also just FYI you can add `/* eslint no-console: 0 */` at the beginning of the file so that you don't have to comment multiple lines.
Of course these are not important changes, it was just for fun